README.md

```diff
@@ -40,7 +40,8 @@ This repository defines a set of accessible themes according to [WCAG color stan
 ## Themes
 
 - [Pitaya Smoothie](./packages/pitayasmoothie/README.md) - Color-blind friendly
-- [GitHub light](./packages/githublight/README.md) - Color-blind friendly
+- [GitHub Light](./packages/githublight/README.md) - Color-blind friendly
+- [Github Dark](./packages/githubdark/README.md) - Color-blind friendly
```

Comment:
Is this so? There are both GH dark and GH dark colourblind friendly https://github.com/primer/github-vscode-theme if the latter then it seems the name of the theme should be corrected

Reply:
In the VSCode documentation it states that all the themes support different types of color blindness, it is not specific to which one though 🤷‍♀️

Do you prefer to only add the `color-blind friendly` note to the themes that are named specifically?

Reply:
Unless a theme is guaranteed to be colourblind-friendly, we should not say it is (otherwise, folks will expect this to be the case).

From the VSCode extension, https://github.com/primer/github-vscode-theme#install, some themes are colourblind friendly - if in this PR you *are not* implementing any of those marked as colourblind friendly, then this should not be called out. (my understanding was you were adding the light and dark default ones)

> GitHub Light Default
GitHub Light High Contrast ✨ new ✨
GitHub Light Colorblind ✨ new ✨
GitHub Dark Default
GitHub Dark High Contrast
GitHub Dark Colorblind ✨ new ✨
GitHub Dark Dimmed

Reply:
ok! I'll make the change